pilot/pkg/serviceregistry/kube/controller.go

```diff
@@ -812,8 +812,13 @@ func (c *Controller) updateEDS(ep *v1.Endpoints, event model.Event) {
 
 	// TODO: Endpoints include the service labels, maybe we can use them ?
 	// nodeName is also included, not needed
-
-	log.Infof("Handle EDS endpoint %s in namespace %s -> %v", ep.Name, ep.Namespace, ep.Subsets)
+	addresses := []string{}
```

Comment:
We're only doing this for the log ... maybe surround with an if-block for the log level?

Reply:
Good idea, fixed this